### ISSUE TITLE
[BUG] Import Inochi Session Data does not import animation tracking data

### ISSUE THREAD

**Issue by Toyoyo** [Inochi2D/inochi-creator#460]
### Validations

- [X] I have checked for similar bug reports and could not find any.
- [X] I have tested and confirmed that this is an issue in an official branded build.

### Describe the bug

When using the option to merge INP Data from Inochi Session into a project, regular tracking settings are imported successfully, but animation tracking isn't.
It only is when using "Import -> Inochi2D Puppet".

### Reproduction

1. Setup Animation Tracking Parameter in Session
2. Save to File
3. Open INX Project in Inochi Creator
4. Select "Tools -> Import Inochi Session Data" to import tracking data.
5. Export to Inochi2d Puppet
6. Open exported file in Inochi Session
7. Animation tracking Parameter is lost (Trigger: Tracking isn't, though)

### System Architecture

x86_64

### Operating System

Windows

### Version

0.8.7

**Comment by grillo-delmal** [comment]
Yeah... I think that I never implemented this in Creator since I only added the animation triggering feature in Session.
I can send a PR to fix this since it's not a hard thing to do.

**Comment by Toyoyo** [comment]
Yeah, it's basically just adding one line in mainmenu.d:
```
diff -rpuN inochi-creator/source/creator/widgets/mainmenu.d inochi-creator.fix/source/creator/widgets/mainmenu.d
--- inochi-creator/source/creator/widgets/mainmenu.d	2024-11-29 02:58:24.574071814 +0100
+++ inochi-creator.fix/source/creator/widgets/mainmenu.d	2024-11-27 01:16:07.379931658 +0100
@@ -426,6 +426,7 @@ void incMainMenu() {
 
                             if ("com.inochi2d.inochi-session.bindings" in p.extData) {
                                 incActivePuppet().extData["com.inochi2d.inochi-session.bindings"] = p.extData["com.inochi2d.inochi-session.bindings"].dup;
+                                incActivePuppet().extData["com.inochi2d.inochi-session.animations"] = p.extData["com.inochi2d.inochi-session.animations"].dup;
                                 incSetStatus(_("Successfully overwrote Inochi Session tracking data..."));
                             } else {
                                 incDialog(__("Error"), _("There was no Inochi Session data to import!"));
```

Thanks for your help building the software.

**Comment by grillo-delmal** [comment]
Fixed in #462
